be/src/column/column_hash.h

```diff
@@ -227,10 +227,12 @@ class SliceHashWithSeed<PhmapSeed2> {
 
 #if defined(__SSE2__) && !defined(ADDRESS_SANITIZER)
 
-// NOTE: This function will access 15 excessive bytes after p1 and p2.
+// NOTE: This function will access 15 excessive bytes after p1 and p2, which should has padding bytes when allocating
+// memory. So it is unsafe, carefully to use it.
 // NOTE: typename T must be uint8_t or int8_t
 template <typename T>
-typename std::enable_if<sizeof(T) == 1, bool>::type memequal(const T* p1, size_t size1, const T* p2, size_t size2) {
+typename std::enable_if<sizeof(T) == 1, bool>::type memequal_unsafe(const T* p1, size_t size1, const T* p2,
```

Comment:
Why to change the name? The annotation is enough if you describe why it's not safe and cautiousness.

Reply:
we found it was misued even if there are annotations.

Reply:
If you want to change, then I suggest name like `memequal_with_padding`. unsafe is still not clear enough.

Reply:
rename it to `memequal_padded()`